### PR TITLE
NAAS-160 - Provide documentation of example JPA + CRDB notary configuration

### PR DESCRIPTION
**Pull request by AaronForinton**

### CODE REVIEW

**Review by benpesterr3 — APPROVED** [review]
LGTM

**Comment by AaronForinton** [comment]
> LGTM

Thank you @benpesterr3. SME approval from @relyafi probably a good idea before merging.

**Review by relyafi — DISMISSED** [review]
LGTM. One minor observation - the example configuration now contains the Cockroach DB specific recommendations. Might be worth tweaking "The configuration below..." text to make this clear, or not including the extra settings in the example.

File: content/en/platform/corda/4.8/enterprise/notary/installing-the-notary-service.md [hunk 1]
```diff
@@ -15,15 +15,17 @@ weight: 2
 
 In configuring the notary worker nodes, please note the following:
 
-
 * The X500 name of the notary service is configured in `notary.serviceLegalName`.
 * Put the IP address or host name of the nearest shared DB server first in the JDBC
-URL. When running a DB node and a notary worker node on a single machine, list the
-local IP first
+  URL. When running a DB node and a notary worker node on a single machine, list the
+  local IP first
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/notary/installing-the-notary-service.md:21]:
```suggestion
  local IP first.
```

File: content/en/platform/corda/4.8/enterprise/notary/installing-the-notary-service.md [hunk 2]
```diff
@@ -15,15 +15,17 @@ weight: 2
...
 * In addition to the connection to the shared DB holding the notary state,
-each notary worker needs to have access to its own local node DB. See the
-*dataSourceProperties* section in the configuration file
+  each notary worker needs to have access to its own local node DB. See the
+  *dataSourceProperties* section in the configuration file
```
Comment (benpesterr3) [inline content/en/platform/corda/4.8/enterprise/notary/installing-the-notary-service.md:24]:
```suggestion
  *dataSourceProperties* section in the configuration file.
```

**Comment by AaronForinton** [comment]
> 2 missing fullstops

Thanks @benpesterr3 fixed now :)

**Review by relyafi — APPROVED** [review]

**Review by benpesterr3 — APPROVED** [review]

**Review by benpesterr3 — APPROVED** [review]

**Pull request merged**